### PR TITLE
XlsxWriter,pdfminer,qimage2ndarry: Python updates

### PR DESCRIPTION
**Pull request by eborisch**
Maintainer updates.

### CODE REVIEW

File: python/py-XlsxWriter/Portfile [hunk 1]
```diff
@@ -4,11 +4,11 @@ PortSystem          1.0
...
 PortGroup           github 1.0
 
-github.setup        jmcnamara XlsxWriter 1.2.5 RELEASE_
+github.setup        jmcnamara XlsxWriter 1.2.8 RELEASE_
 name                py-XlsxWriter
 
-python.versions     27 35 36 37 38
-python.default_version 27
+python.versions     27 36 37 38
+python.default_version 38
```
Comment (mf2k) [inline python/py-XlsxWriter/Portfile:11]: I think that the `python.default_version` line should be removed so the port defers to the default in the python portgroup.

File: python/py-pdfminer/Portfile [hunk 2]
```diff
@@ -2,9 +2,14 @@
...
 PortGroup           python 1.0
-
+PortGroup           github 1.0
+github.setup        euske pdfminer 423f851
+version             20200118
 name                py-pdfminer
-version             20131113
+
+python.versions         36 37 38
+python.default_version  38
```
Comment (mf2k) [inline python/py-pdfminer/Portfile:11]: Same comment as above. Please consider removing this line.

File: python/py-pdfminer/Portfile [hunk 3]
```diff
@@ -2,9 +2,13 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
-
+PortGroup           github 1.0
+github.setup        euske pdfminer 423f851
+version             20200118
```
Comment (mf2k) [inline python/py-pdfminer/Portfile:7]: I think putting dashes in there makes it clearer that this is a date. Also that's how most other ports handle it. 

`version             2020-01-18`

File: python/py-pdfminer/Portfile [hunk 4]
```diff
@@ -2,9 +2,13 @@
...
 name                py-pdfminer
-version             20131113
+
+python.versions         36 37 38
+
```
Comment (mf2k) [inline python/py-pdfminer/Portfile:11]: Spacing issue here. Make sure there are no tabs here.

File: python/py-pdfminer/Portfile (hunk 3 above)
Comment (eborisch) [reply python/py-pdfminer/Portfile:7]: By a quick grep, 20 ports use YYYY-MM-DD, while 210 use YYYYMMDD.

Add to this the fact that this port currently uses YYYYMMDD, I plan to stick with that, unless there is some other compelling reason to change.

Comment (mf2k) [reply python/py-pdfminer/Portfile:7]: Sounds good.

**Pull request merged**